addons/index.md

```diff
@@ -7,16 +7,109 @@ layout: documentation
-| [Persistence](persistence.html) | Persistence services allow openHAB to store time series data for history-based actions or statistics |
-| [Actions](actions.html) | Actions are predefined methods for openHAB rules and scripts |
-| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for items |
-| [Voice Services](voices.html) | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc. |
-| [3rd Party System Integration](io.html) | Expose openHAB to external systems |
+This includes all 2.x Bindings as well as all 1.x add-ons that were reported to be compatible.
+There are several ways you can install an add-on. 
+These are described under *Installation of Add-ons* below
+
+| Add-on Type                             | Description                                                                                                               |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| [Bindings](bindings.html)               | Bindings integrate physical hardware, external systems and web services in openHAB                                        |
+| [User Interfaces](uis.html)             | User interfaces are either native smartphone apps or web applications that access the openHAB server through the REST API |
+| [Persistence](persistence.html)         | Persistence services allow openHAB to store time series data for history-based actions or statistics                      |
+| [Actions](actions.html)                 | Actions are predefined methods for openHAB rules and scripts                                                              |
+| [Transformations](transformations.html) | Transformations are used to translate between technical and human-readable values for Items                               |
+| [Voice Services](voices.html)           | Services that provide voice enabling features, such as text-to-speech, speech-to-text etc.                                |
+| [3rd Party System Integration](io.html) | Expose openHAB to external systems                                                                                        |
+
+## Installation of Add-ons
+
+Depending on the [package]({{base}}/configuration/packages.html) you have choosen during your first time setup, there are already some pre-installed add-ons.
+Additional add-ons can be installed in the different ways, described below.
+
+### Through Paper UI
+
+Navigate to the add-ons section.
+Search for the desired add-on in the categories and press install.
+
+![installing Add-ons through Paper UI](images/index_installation_paperui.gif)
+
+### Through Configuration Files
+
+For this installation option you need to know the `id` of the desired add-on
```

Comment:
Missing final stop

Reply:
Also the example is great but for the user who just quickly needs a hint I'd suggest to give a quick example here.

> ...you need to know the ID of the desired Add-on, e.g., `network` or `mqtt1`.